grassroots-shared/src/util/DeepLog.ts

```diff
@@ -0,0 +1,13 @@
+const replacer: (this: unknown, key: string, value: unknown) => unknown = (
```

Comment:
I keep trying to Log maps and having it break.
I think landing this helper is probably reasonable, even though I'm not actually using it in the code landed in this patch.